pex/bin/pex.py

```diff
@@ -926,11 +928,12 @@ def walk_and_do(fn, src_dir):
 
 
 def transform_legacy_arg(arg):
+    # type: (str) -> str
     # inherit-path used to be a boolean arg (so either was absent, or --inherit-path)
     # Now it takes a string argument, so --inherit-path is invalid.
     # Fix up the args we're about to parse to preserve backwards compatibility.
     if arg == "--inherit-path":
-        return "--inherit-path=prefer"
+        return "--inherit-path={}".format(InheritPath.PREFER.value)
```

Comment:
In retrospect, would've been nice to drop this legacy behaviour in 2.0. Maybe worth deprecating?

Reply:
Agreed, but we still have no deprecation mechanism: #584